packages/in-app-notification-manager/vitest.integration.config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+  test: {
+    globalSetup: ["./test/vitestIntegrationGlobalSetup.ts"],
+    include: ["./test/integration/**/*.test.ts"],
+    testTimeout: 60000,
+    hookTimeout: 60000,
+    fileParallelism: false,
+    pool: "forks",
+    env: { FEATURE_FLAG_SQL: "false" },
```

Comment:
considering this won't have the two different read models, shouldn't we have just one `vitest.integration.config.ts` file (no `integrationsql`) without this feature flag?